__tests__/app.test.js

```diff
@@ -91,4 +95,30 @@ describe('GET /api/topics', () => {
         });
         });
       });
-    });
\ No newline at end of file
+    });
+
+//TASK 6
+describe('GET /api/:article_id/comments', () => {
+  test('200: serves an array of all comments for the given article_id', () => {
+    return request(app)
+    .get('/api/articles/3/comments')
+    .expect(200)
+    .then(({body}) => {
+      const { comments } = body;
+      expect(Array.isArray(comments)).toBe(true);
+      expect(comments).toHaveLength(2);
+      comments.forEach((comment) => {
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: 3
+          })
+        );
+      });
+      });
+    });
```

Comment:
Really nice first test, but there are some missing tests for this endpoint:

What if a user tried to search for comments on an article that had no comments? 
E.g `/api/articles/2/comments` (I believe this one has no comments, but feel free to double check the test data)

What if a user tried to find comments for an article that didnt exist? 
E.g `/api/articles/999/comments`

What if a user tried to find comments for an article using an invalid article_id? 
E.g. `/api/articles/banana/comments`